rbac/management/role/relation_api_dual_write_handler.py

```diff
@@ -301,13 +302,17 @@ def _replicate(self):
             self._replicator.replicate(
                 ReplicationEvent(
                     event_type=self.event_type,
-                    info={"v1_role_uuid": str(self.role.uuid)},
+                    info={
```

Comment:
Can you also add ids of self.binding_mappings ?  (safely, if they exists, if they empty,.. )